src/main/java/nextstep/courses/domain/session/SessionType.java

```diff
@@ -4,4 +4,11 @@ public enum SessionType {
     FREE,
     PAID;
 
+    public static boolean isFree(SessionType sessionType) {
+        return sessionType == FREE;
+    }
```

Comment:
```suggestion
    public boolean isFree() {
        return this == FREE;
    }
```
위와 같이 인스턴스 메서드로 구현하는 것은 어떨까?